classes/wordpress.php

```diff
@@ -1082,9 +1082,12 @@ private function user_delete( $id, $reassign = null ) {
 	 *     success: 1
 	 *   "errors" : [ ],
 	 */
-	private function post_create( $params, $id_field = 'ID', $post_type = '' ) {
```

Comment:
I think we can leave the default here, since `wp_insert_post` assumes the type is `post` if there is not a different value, but aside from that I'll go ahead and merge this and then put it on the other methods, as in the other PR.

Reply:
Nevermind I see why this is here.